zzk/service/hoststate.go

```diff
@@ -219,8 +219,16 @@ func (l *HostStateListener) Spawn(cancel <-chan interface{}, stateID string) {
 			return
 		}
 
+		// set up a listener on container state
```

Comment:
What does this comment mean?  It looks like you are just getting the container state.  You aren't listening to it.